gulpfile.js

```diff
@@ -1,88 +1,183 @@
+   * Apply the namespace on html and body elements
+   *
+   * @param  {string} selector the current selector to be transformed
+   * @return string
+   */
+  function namespaceRootElements (selector) {
+    const regex = /^(body|html)/;
+
+    if (regex.test(selector)) {
+      selector = selector.replace(regex, match => {
+        return match + bootstrapNamespace;
+      }) + ',\n' + selector.replace(regex, bootstrapNamespace);
+    }
+
+    return selector;
   }
 
-  return selector;
+  /**
+   * Deletes the special class that was used as marker for styles that should
+   * not be nested inside the bootstrap namespace from the given selector
+   *
+   * @param  {String} selector
+   * @return {String}
+   */
+  function removeOutsideNamespaceMarker (selector) {
+    return selector.replace(outsideNamespaceRegExp, '');
+  }
 }
 
-/**
- * Deletes the special class that was used as marker for styles that should
- * not be nested inside the bootstrap namespace from the given selector
- *
- * @param  {String} selector
- * @return {String}
- */
-function removeOutsideNamespaceMarker (selector) {
-  return selector.replace(outsideNamespaceRegExp, '');
+// release
+{
+  const git = require('simple-git/promise')(__dirname);
+  const replace = require('gulp-replace');
+
+  gulp.task('release:create-branch', () => {
+    return git.checkoutLocalBranch(releaseBranchName());
+  });
+
+  gulp.task('release:update-info', () => {
+    return gulp.src(['info.xml'])
+      .pipe(replace(/<version>([^>]+)<\/version>/g, `<version>${argv.ver}</version>`))
+      .pipe(replace(/<releaseDate>([^>]+)<\/releaseDate>/g, `<releaseDate>${formatTodayDate()}</releaseDate>`))
+      .pipe(gulp.dest('.'));
+  });
+
+  gulp.task('release:commit-changes', () => {
+    return git.add(['info.xml', '*.css'])
+      .then(() => {
+        return git.commit(`Update css and info files for v${argv.ver}`, null, {
+          '--no-verify': null
+        });
+      });
+  });
+
+  gulp.task('release:params-check', done => {
+    if (typeof argv.ver === 'undefined') {
+      throw new PluginError('release', {
+        message: 'Please specify the version using --ver (example: "--ver 1.1.0")',
+        showProperties: false
+      });
+    }
+
+    done();
+  });
+
+  gulp.task('release:push', () => {
+    return git.push(['-u', 'origin', releaseBranchName()]);
+  });
+
+  gulp.task('release', gulp.series(
+    'release:params-check',
+    'release:create-branch',
+    'sass',
+    'release:update-info',
+    'release:commit-changes',
+    'release:push',
+    function releasePrepared (done) {
+      const compareUrl = `https://github.com/civicrm/org.civicrm.shoreditch/compare/${releaseBranchName()}`;
+
+      console.log(`Branch ${releaseBranchName()} prepared and pushed upstream`);
+      console.log(`Please check ${compareUrl}`);
+      done();
+    }
+  ));
+
+  /**
+   * Build the YYYY-MM-DD string of today's date
+   *
+   * @return {String}
+   */
+  function formatTodayDate () {
```

Comment:
Any reason we do not use *MomentJS* here?

Reply:
I guess http://tonsky.me/blog/disenchantment/ hit home to me, I just didn't want to add a package as complex as momentjs just for a one-off so trivial